ApiBundle/Controller/ContentController.php

```diff
@@ -263,18 +263,24 @@ protected function findOneContent($contentId, $language, $version = null)
     /**
      * @param boolean|null $published
      *
-     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_content_list_author_not_published", defaults={"published": false})
-     * @Config\Route("/list/by-author", name="open_orchestra_api_content_list_author", defaults={"published": null})
+     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_content_list_author_and_site_not_published", defaults={"published": false})
```

Comment:
I think API methods should be generic, so I not agree with giving to routes a name related to a specific view.